IPython/html/static/base/js/page.js

```diff
@@ -10,6 +10,7 @@ define([
 
     var Page = function () {
         this.bind_events();
+        this.show_anchor_link();
```

Comment:
I think this is the source of the scroll problems on load—it's firing too early. If this is called after the notebook finishes loading (`'notebook_loaded.Notebook` event), I think it might be okay.